<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_le">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;    
	background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 5px; /* Закругление углов (по желанию) */
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_le">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 5px; /* Закругление углов (по желанию) */
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_le">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 5px; /* Закругление углов (по желанию) */
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_main_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QLabel" name="reg_res">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('data/textures/background.jpg');
background-repeat: no-repeat; 
background-position: center;</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>email_le</zorder>
   <zorder>password_le</zorder>
   <zorder>username_le</zorder>
   <zorder>to_main_btn</zorder>
   <zorder>reg_btn</zorder>
   <zorder>reg_res</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
